action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -547,6 +563,8 @@ func (stateDB *StateDBAdapter) Prepare(rules params.Rules, sender, coinbase comm
 	if rules.IsShanghai { // EIP-3651: warm coinbase
 		stateDB.AddAddressToAccessList(coinbase)
 	}
+	// Reset transient storage at the beginning of transaction execution
+	stateDB.transientStorage = newTransientStorage()
```

Comment:
same thing, not necessary

Reply:
it is best to add

Reply:
as discused, let's remove this, it is done in `NewStateDBAdapter`